TrackletAlgorithm/MatchCalculator.h

```diff
@@ -818,13 +1189,21 @@ void MatchCalculator(BXType bx,
     bool goodmatch_next = false;
 
     // For first tracklet, pick up the phi cut value
+    best_delta_z = (newtracklet)? LUT_matchcut_z[proj_seed] : best_delta_z;
     best_delta_phi = (newtracklet)? LUT_matchcut_phi[proj_seed] : best_delta_phi;
 
     // Check that matches fall within the selection window of the projection
     if ((delta_z_fact < LUT_matchcut_z[proj_seed]) && (delta_z_fact >= -LUT_matchcut_z[proj_seed]) && (abs_delta_phi <= best_delta_phi)){
       // Update values of best phi parameters, so that the next match
       // will be compared to this value instead of the original selection cut
-      best_delta_phi = abs_delta_phi;
+      if(isDisk) {
+        best_delta_rphi = isPSStub ? ap_uint<20>(abs_delta_phi * ap_uint<12>(stub_ps_r)) : ap_uint<20>(abs_delta_phi * tmp_stubr);
```

Comment:
Do you really need two branches here?Is tmp_stubr not set to the radius of the stub for both ps and 2s stubs?

Reply:
`tmp_stubr` was only used for the 2S. I renamed it to `disk_stubr2s`